zdm/cube.py

```diff
@@ -1,4 +1,4 @@
-
+#WARNING - THIS ROUTINE USES OUTDATED NOTATION AND NEEDS TO BE UPDATED
```

Comment:
I'd say this file is fully DEPRECATED

Reply:
Yes, but wanted to make this explicit. CLearly to be removed at some point, e.g. in the cleanup once we merged back to main before the paper comes out.